src/Symfony/Component/HttpFoundation/Tests/Test/Constraint/ResponseStatusCodeSameTest.php

```diff
@@ -48,9 +46,8 @@ public function testReducedVerbosity()
         try {
             $constraint->evaluate(new Response('Response body', 404));
         } catch (ExpectationFailedException $e) {
-            $exceptionMessage = TestFailure::exceptionToString($e);
-            $this->assertStringContainsString("Failed asserting that the Response status code is 200.\nHTTP/1.0 404 Not Found", TestFailure::exceptionToString($e));
-            $this->assertStringNotContainsString('Response body', $exceptionMessage);
+            $this->assertStringContainsString("Failed asserting that the Response status code is 200.\nHTTP/1.0 404 Not Found", $e->getMessage());
+            $this->assertStringNotContainsString('Response body', $e->getMessage());
```

Comment:
You didn't cause this, but this test looks flawed. It passes if no exception is raised and I don't believe that this is intended.

Reply:
see #58178